app-connector-client/server/app.js

```diff
@@ -37,12 +37,6 @@ function init(varkesConfigPath = null, currentPath = "", nodePortParam = null) {
     app.use(CONNECTION, connector.router(varkesConfig, nodePortParam))
     app.use(EVENTS_URL, events.router())
 
-    app.set('view engine', 'ejs')
```

Comment:
Have you removed the ejs dependency in package.json?

Reply:
removed it